File: ReturnOrder.java
public class ReturnOrder {
    private int regUserFlag;
    private int id;
    private double price;
    private String address;
    private ArrayList<Book> booksSold;

    // Constructor

    public ReturnOrder(int regUserFlag, int id, double price, String address, ArrayList<Book> booksSold) {
        this.regUserFlag = regUserFlag;
        this.id = id;
        this.price = price;
        this.address = address;
        this.booksSold = booksSold;
    }

    // Getters and Setters

    public int getRegUserFlag() {
        return regUserFlag;
    }

    public void setRegUserFlag(int regUserFlag) {
        this.regUserFlag = regUserFlag;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public double getPrice() {
        return price;
    }

    public void setPrice(double price) {
        this.price = price;
    }

    public String getAddress() {
        return address;
    }

    public void setAddress(String address) {
        this.address = address;
    }

    public ArrayList<Book> getBooksSold() {
        return booksSold;
    }

    public void setBooksSold(ArrayList<Book> booksSold) {
        this.booksSold = booksSold;
    }
}

File: model/ReturnOrder.php
<?php
class ReturnOrder {
    private $regUserFlag;
    private $id;
    private $price;
    private $credit;
    private $booksReturned = array();

    // Constructor

    public function __construct(&$regUserFlag, &$id, &$price, &$credit, &$booksReturned) {
        $this->regUserFlag = $regUserFlag;
        $this->id = $id;
        $this->price = $price;
        $this->credit = $credit;
        $this->booksReturned = $booksReturned;
    }
    // Getters and Setters

    public function getRegUserFlag() {
        return $this->regUserFlag;
    }

    public function setRegUserFlag(&$regUserFlag) {
        $this->regUserFlag = $regUserFlag;
    }

    public function getId() {
        return $this->id;
    }

    public function setId(&$id) {
        $this->id = $id;
    }

    public function getPrice() {
        return $this->price;
    }

    public function setPrice(&$price) {
        $this->price = $price;
    }

    public function getCredit() {
        return $this->credit;
    }

    public function setCredit(&$credit) {
        $this->credit = $credit;
    }

    public function getBooksReturned() {
        return $this->booksReturned;
    }

    public function setBooksReturned(&$booksReturned) {
        $this->booksReturned = $booksReturned;
    }

    public function addBook(&$book){
        array_push($this->booksReturned, $book);
    }
}

?>
